contrib/babelfishpg_tsql/sql/information_schema_tsql.sql

```diff
@@ -717,3 +717,28 @@ CREATE OR REPLACE VIEW information_schema_tsql.routines AS
 GRANT SELECT ON information_schema_tsql.routines TO PUBLIC;
 
 SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false);
+
+
+CREATE OR REPLACE VIEW information_schema_tsql.domain_constraints
+AS SELECT sys.db_name()::information_schema.sql_identifier AS constraint_catalog,
```

Comment:
Instead of directly printing sys.db_name() - we should add that as condition to filter out the query result.
i.e. babelfish_namespace_ext.dbid = sys.db_id()
Please refer existing information_schema_tsql views for reference.